src/main/java/Duke.java

```diff
@@ -1,10 +1,232 @@
+                case ("T"):
+                    task = new ToDo(readTask[2]);
+                    break;
+                case ("D"):
+                    task = new Deadline(readTask[2], readTask[3]);
+                    break;
+                case ("E"):
+                    task = new Event(readTask[2], readTask[3]);
+                    break;
+                default:
+                    task = new Task(readTask[2]);
+                }
+
+                if (readTask[1].equals("true")){
+                    task.setIsDone(true);
+                }
+                tasks.add(task);
+            }
+        } catch (FileNotFoundException e){
+            System.out.println("File not found");
+        } catch (IOException e){
+            e.printStackTrace();
+            System.out.println("File creation failed");
+        }
+    }
+
+    //print the horizontal line
+    public static void printHorizontalLine() {
+        System.out.println(STRAIGHT_LINE);
+    }
+
+    //print a welcome message
+    public static void printWelcomeMessage() {
+        printHorizontalLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        printHorizontalLine();
+    }
+
+    //print a goodbye message
+    public static void printByeMessage() {
+        printHorizontalLine();
+        System.out.println("Bye. Hope to see you again soon!\n");
+        printHorizontalLine();
+    }
+
+    //echo the message
+    public static void echoMessage(Task message, int index) {
+        printHorizontalLine();
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(message.toString());
+        System.out.println("Now you have " + index + " tasks in the list");
+        printHorizontalLine();
+    }
+
+    //echo the list
+    public static void echoList(ArrayList<Task> tasks) {
+        printHorizontalLine();
+        int index = 0;
+        for (Task task : tasks) {
+            System.out.println((index + 1) + ". " + task.toString());
+            index++;
+        }
+
+        System.out.println();
+        printHorizontalLine();
+    }
+
+    //change the status of the task
+    public static void changeStatus(Task task) {
+        printHorizontalLine();
+        System.out.println("Nice! I've marked this task as done:");
+
+        task.setIsDone(true);
+        System.out.println(task.toString());
+
+        System.out.println();
+        printHorizontalLine();
+    }
+
+    public static Task createType(String message) {
+        String[] infoEntered = message.split(" ");
+        int nameLength;
+        try {
+            switch (infoEntered[0]) {
+            case ("todo"):
+                nameLength = 4;
+                if (infoEntered.length == 1) {
+                    throw new DukeException();
+                }
+                return new ToDo(message.substring(nameLength + 1));
+            case ("deadline"):
+                nameLength = 8;
```

Comment:
magic number? please refer to: https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#guideline-name-well